### PR TITLE
Bug 1301037 - Add option to choose EMR release

### PR DESCRIPTION
**Pull request by maurodoglio**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-analysis-service/14)

### CODE REVIEW

File: atmo/clusters/forms.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from django import forms
+from django.conf import settings
```
Comment (jezdez) [inline atmo/clusters/forms.py]: Don't think that's needed

File: atmo/clusters/models.py [hunk 2]
```diff
@@ -36,6 +40,13 @@ class Cluster(models.Model):
         max_length=50, blank=True, null=True,
         help_text="AWS cluster/jobflow ID for the cluster, used for cluster management."
     )
+
+    emr_release = models.CharField(
+        max_length=50, choices=zip(*(EMR_RELEASES,)*2), default=EMR_RELEASES[-1],
```
Comment (jezdez) [inline atmo/clusters/models.py]: `zip` returns a `zip` object on Python 3, not sure if the choices parameter handles that nicely. wrapping it in a `list` would appease my mind.

Side note: did we ever want to make atmo Python 3 btw?

File: atmo/clusters/models.py [hunk 3]
```diff
@@ -7,7 +7,11 @@
 from ..utils import provisioning
 
 
+EMR_RELEASES = ('5.0.0', '4.5.0')
```
Comment (jezdez) [inline atmo/clusters/models.py]: Maybe add a comment here that the last one will be the default?

File: atmo/templates/atmo/new-cluster.jinja [hunk 4]
```diff
@@ -33,6 +33,13 @@
           <p class="help-block"></p>
         </div>
         <div class="alert alert-warning">Upload your SSH <strong>public key</strong>, not private key! This will generally be found in places like <code>~/.ssh/id_rsa.pub</code>.</div>
+        <div class="form-group">
+          <label>EMR release</label>
+          <div class="control-group">
+            {{ new_cluster_form.emr_release }}
```
Comment (jezdez) [inline atmo/templates/atmo/new-cluster.jinja]: I only now realize we don't print any form errors in that template, any reason not to?

File: atmo/clusters/models.py (hunk 2 above)
Comment (maurodoglio) [inline atmo/clusters/models.py]: We have never discussed that tbh. Let's have a chat about it.

File: atmo/templates/atmo/new-cluster.jinja (hunk 4 above)
Comment (maurodoglio) [inline atmo/templates/atmo/new-cluster.jinja]: I would rewrite the whole form template tbh. There are a few things there that break the DRY principle at the moment. Let's talk about it.

File: atmo/clusters/models.py (hunk 3 above)
Comment (maurodoglio) [inline atmo/clusters/models.py]: Good idea

**Comment by maurodoglio** [comment]
This pull request has already been merged

**Pull request closed without merging**